app/resources/responders.html.template

```diff
@@ -0,0 +1,72 @@
+{% extends "static-base.html.template" %}
+{% load i18n %}
+
+{% block content %}
+<h1>Information for responders</h1>
+
+Any time Google launches Google Person Finder in response to a crisis,
+government and nonprofit organizations can help people locate each other
+by spreading the word, using the data, and adding information.
+
+<div class="section">
+<img class="inset" src="{{env.global_url}}/embed-80x57.png"
+    width="80" height="57" alt="Embed Person Finder">
+<h2>Embed Google Person Finder in your site</h2>
+
+<p>
+Spread the word about Google Person Finder to your community.
+You can point people to
+<a href="http://google.org/personfinder">this Google Person Finder page</a>,
+or you can embed a small version of Google Person Finder
+directly on your website using the following HTML code:
+
+<pre>
+&lt;iframe
+    src="http://google.org/personfinder/<var>repository</var>/?ui=small"
+    width=400 height=300 frameborder=0
+    style="border: dashed 2px #77c"&gt;&lt;/iframe&gt;
+</pre>
+
+<p>
+(Replace <var>repository</var> with the appropriate repository name.)
+
+<p>
+The above gadget code is made available under the Apache 2.0 license,
+and any data you receive in connection with the gadget
+is shared under a Creative Commons license.
+</div>
+
+<div class="section">
+<img class="inset" src="{{env.global_url}}/download-80x99.png"
```

Comment:
Again, I want to keep this pull request simple to just submit the resources already in production as is. I can try that later. Same for other files.